beacon-chain/rpc/beacon/slashings.go

```diff
@@ -55,7 +55,11 @@ func (bs *Server) SubmitAttesterSlashing(
 			return nil, err
 		}
 	}
-	slashedIndices := sliceutil.IntersectionUint64(req.Attestation_1.AttestingIndices, req.Attestation_2.AttestingIndices)
+	indices := sliceutil.IntersectionUint64(req.Attestation_1.AttestingIndices, req.Attestation_2.AttestingIndices)
```

Comment:
For `Slots` I was defining `sliceutil.IntersectionSlot()` method which returned typed intersection. So, for consistency, shouldn't we define `sliceutil.IntersectionValidatorIndex()` (the best we can have before generics, lol). Then here it will be just `sliceutil.IntersectionUint64 -> sliceutil.IntersectionValidatorIndex`).

Reply:
Well. `AttestingIndices` is still `[]uint64` given the lack of fssz support. We'll work on that next. 

There's not much use for `IntersectionValidatorIndex` now. I do agree once we can extend support to `AttestingIndices` to use `types.ValidatorIndex`, then `IntersectionValidatorIndex` will be needed